src/SDL_internal.h

```diff
@@ -28,7 +28,14 @@
 
 /* Need this so Linux systems define fseek64o, ftell64o and off64_t */
 #ifndef _LARGEFILE64_SOURCE
+#ifndef __QNX__
 #define _LARGEFILE64_SOURCE
```

Comment:
Looking for feedback on this change. Is `_LARGEFILE_SOURCE` only needed for Linux?

Reply:
I think it might be used on BSD as well. Is it causing problems on QNX?